tradeexecutor/strategy/interest.py

```diff
@@ -94,7 +94,7 @@ def update_credit_supply_interest(
 
     # Update interest stats
     interest.last_accrued_interest = position.calculate_accrued_interest_quantity()
-    interest.last_updated_at = datetime.datetime.utcnow()
+    interest.last_updated_at = event_at
```

Comment:
- `updated_at` is trade-executor wall clock time by the spec
- `event_at` is blockchain time